src/fetchServices.ts

```diff
@@ -81,30 +54,18 @@ export async function listServices(
  * @param serviceName - the hostname of a service ('foo.googleapis.com')
  * @return the service config of the passed hostname
  */
-export async function getServiceConfig(
-  serviceName: string
-): Promise<servicemanagement_v1.Schema$Service> {
-  const res = await servicemanagement.services.getConfig(
-    {
-      serviceName: serviceName,
-    },
-    {
-      retryConfig: {
-        retry: 10,
-      },
-    }
-  );
-  return res.data;
+export async function getServiceConfig(serviceName: string) {
```

Comment:
Need to retry on `RESOURCE_EXHAUSTED` or, you know, get more quota :)

Reply:
Which project is this using?  I'd rather us apply for the quota increase rather than hit it over the head with retries :) Alternatively - we could unroll the loop to not make requests in parallel, and do them synchronously?